apps/client/src/app/pages/portfolio/allocations/allocations-page.component.ts

```diff
@@ -42,10 +42,12 @@ export class AllocationsPageComponent implements OnDestroy, OnInit {
   public sectors: {
     [name: string]: { name: string; value: number };
   };
+  public symbols: {
+    [name: string]: { name: string; value: number };
+  };
   public user: User;
 
   private unsubscribeSubject = new Subject<void>();
-
```

Comment:
Why do you remove this line break? 😄

Reply:
Just playing around! I also ran the linters so the tests could pass. I'm putting it back :-)